src/style.css

```diff
@@ -1,3 +1,68 @@
+:root {
+  --link-color: #00ACDC;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.nav {
+  padding: 0;
+}
+
+.logo-img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  list-style: none;
   margin: 0;
 }
+
+
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 60px;
+  text-decoration: none;
+  margin-left: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children